### PR TITLE
Add Cairo Graphics icon

### PR DESCRIPTION
**Pull request by s-kond**
cairographics

To create the svg-icon, I used inkscape and a svg-file from the github-repository that was linked in the issue:
https://github.com/freedesktop/cairo/blob/44f808fce9f437e14f2b0ef4e1583def8ab578ae/doc/tutorial/slides/cairo-title.svg

There were two primary colors:
     - the blue background ("#162284")
     - and the yellow/orange scarabaeus ("#f39914")

I picked the yellow/orange "#f39914", because the blue color is just used for the background - but I wasn't sure about that.

### CODE REVIEW

**Comment by dirien** [comment]
Hi @s-kond,

thanks for the contribution! Could you check the lint error so I can review your PR? Thanks.

**Comment by s-kond** [comment]
Hi @dirien,
I checked the lint error and it shows me this: 

```
Data file is formatted incorrectly:
   {
       "icons": [
@@ 12469 more lines
       ]
   }
+
```
So there should be a formatting error, but I don't know what's wrong or how I can solve this... Can you help me?

**Comment by dirien** [comment]
Maybe and Windows / Linux CR/LF issue?

File: _data/simple-icons.json [hunk 1]
```diff
@@ -12476,4 +12481,4 @@
             "source": "https://www.zyte.com/"
         }
     ]
-}
+}
```
Comment (mondeja) [inline _data/simple-icons.json:12484]: You need to add a newline character here:

```suggestion
}

```

**Review by mondeja — CHANGES_REQUESTED** [review]

**Comment by s-kond** [comment]
Simply adding a new line at the end solved the problem, thx!
But it seems I have to change some more things - I'm working on that

**Review by dirien — CHANGES_REQUESTED** [review]
HI @s-kond,

I see some little differences

image

Maybe try up the precision to 5?

**Review by dirien — APPROVED** [review]
Hi @s-kond,

Thanks for your contribution to simple-icons!

I think it's a very close fit!

**Pull request merged**